src/main.rs

```diff
@@ -222,7 +224,14 @@ fn elf_to_tbf(
         // is, then we need to calculate the correct init_fn_offset.
         if input.ehdr.entry >= section.shdr.addr
             && input.ehdr.entry < (section.shdr.addr + section.shdr.size)
+            && (section.shdr.name.find("debug")).is_none()
         {
+            // panic in case we detect entry point in multiple sections.
+            if entry_point_found {
```

Comment:
Might be nicer to make this a reference to / copy of the first place an entry point was found, that way the error message can print both places entry points were found for easier debugging.

Assuming that's easy, it'd be a nice change; failing that I'm happy to merge this as-is 👍

Reply:
In the verbose mode, we currently print the first entry point, followed by a panic (in the subsequent iteration of the loop). Wouldn't that accomplish the same thing?

```
            // panic in case we detect entry point in multiple sections.
            if entry_point_found {
                panic!("Duplicate entry point in {} section", section.shdr.name);
            }
            entry_point_found = true;

            if verbose {
                println!("Entry point is in {} section", section.shdr.name);
            }
```